services/search/constants.py

```diff
@@ -0,0 +1,5 @@
+SPANISH_API_URL = "http://sesat.fdi.ucm.es:8080/servicios/rest/sinonimos/json/"
+
+KEYWORDS_ENDPOINT = 'http://localhost:8080'
```

Comment:
Instead of having this be a constant, please read this from a local environment variable when your server starts up, so it's possible to switch between local testing and production. You can pass the envvar to your docker container when you run it using the `docker run --env` flag: https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file

(And GKE has a way to pass it to docker too, but you can defer that to whoever is setting up GKE)